<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1106</width>
    <height>955</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_result">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>291</width>
      <height>511</height>
     </rect>
    </property>
    <property name="title">
     <string>结局得分：</string>
    </property>
    <widget class="QCheckBox" name="checkBox_0">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>风雪蔽目(未完成)</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>巍峨银凇(普通一)(50)</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_sp1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>深寒造像(异格一)(150)</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>萨米之熵(普通二)(100)</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_sp2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>虚无之偶(异格二)(120)</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>园丁(普通三)(80)</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_sp3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>哨兵(异格三)(300)</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>时光之沙(普通四)(150)</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_sp4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>迈入永恒(异格四)(250)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_resScore">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>-10</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_more">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>390</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>多隐藏结局(100)</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_triple">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>430</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>击败阿沙放生可汗(99)</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_notree">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>470</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>不进树洞完成结局(200)</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_tmp">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>291</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>临时招募得分：</string>
    </property>
    <widget class="QLabel" name="label_tmp6Word">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>六星(50)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_tmp6Sub">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_tmp5Sub">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>90</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_tmp4Sub">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>140</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_tmp5Word">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>五星(20)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_tmp4Word">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>四星(10)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_tmp6Score">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>40</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_tmp5Score">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>90</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_tmp4Score">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>140</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_tmp4Num">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>140</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_tmp5Num">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>90</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_tmp6Num">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_tmpAllScore">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>-10</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_tmp4Add">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>140</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_tmp5Add">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>90</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_tmp6Add">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>40</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_animal">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>770</y>
      <width>291</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>鸭狗熊、怪物箱得分：</string>
    </property>
    <widget class="QLabel" name="label_animalScore">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>-10</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_animal">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="sliderPosition">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_animalNum">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_animalWord">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>数量：</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_animal">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>81</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
   </widget>
   <zorder>groupBox_tmp</zorder>
   <zorder>groupBox_result</zorder>
   <zorder>groupBox_animal</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1106</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
